src/features/duplicates/hooks/useDuplicatesMutations.tsx

```diff
@@ -11,15 +21,21 @@ export default function useDuplicatesMutations(
   const apiClient = useApiClient();
   const dispatch = useAppDispatch();
 
-  type ZetkinDuplicatePatchBody = Partial<
+  type PotentialDuplicatePatchBody = Partial<
```

Comment:
I don't think this type is accurate, because it means that you can include `duplicates`, `merged` etc in the `PATCH` which you can't. 

I believe the type should just be this:

```ts
type PotentialDuplicatePatchBody = {
  dismissed?: boolean;
};
```